### PR TITLE
Add support for VMware ESXi 7.0 u2 and u3

### PR DESCRIPTION
**Pull request by weizhouapache**

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]
LGTM, let's future proof and add 70u3 too? (At least I see vCenter 7.0u3 is out, if not esxi 7.0u3)

**Comment by rohityadavcloud** [comment]
cc @nvazquez

**Comment by weizhouapache** [comment]
@nvazquez 
could you please review it ?

File: Ansible/tasks/buildvms_custom_allocator.yml [hunk 1]
```diff
@@ -499,6 +499,8 @@
 - name: SSH into VC container and replace vmx file for VC
   shell: "sshpass -p 'P@ssword123' ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@{{ vc_retval.results[0].default_ip }} '{{ item }}'"
   with_items:
+    - "sed -i \"/^ethernet0.address =/d\" $(find -name VC.vmx)"
+    - "sed -i \"/^ethernet0.addressType =/d\" $(find -name VC.vmx)"
```
Comment (nvazquez) [inline Ansible/tasks/buildvms_custom_allocator.yml:503]: Nice, removed in case the strings exist 👍🏼

**Review by nvazquez — APPROVED** [review]
LGTM @weizhouapache

**Comment by weizhouapache** [comment]
thanks @nvazquez @rohityadavcloud 
merging it

**Pull request merged**